python/feathub/feature_tables/sinks/sink.py

```diff
@@ -24,6 +24,7 @@ class Sink(FeatureTable, ABC):
     Base class for all Sink Feature Table.
     """
 
+    # TODO: make sure sink subclasses pass in meaningful table name.
```

Comment:
Can you explain the motivation of this TODO and how we can possibly achieve this?

The `name` field is given by user, similar to how user can define features' names. It is not clear how we can possibly enforce this TODO.

Reply:
Sorry for the ambiguity. Currently Sink implementations have not allowed users to specify the sink table name and would only pass an empty string to the super constructor, for example
```python

class RedisSink(Sink):
    def __init__(
        self,
         ...
    ):
        super().__init__(
            name="",
            system_name="redis",
            table_uri={
                "host": host,
                "port": port,
                "db_num": db_num,
                "namespace": namespace,
            },
            keep_timestamp_field=keep_timestamp_field,
        )
```

The goal of this TODO is to expose the ability to configure sink table name to users. I have updated the TODO to clarify this.